### DISCUSSION TITLE
4.9.0-0.okd-2021-11-28-035710

### DISCUSSION THREAD

**Discussion by emiranda882** [okd-project/okd#1000] [Q&A] [answered]
good my friends, i am problem install okd4 with aws error -> FATAL failed to fetch Cluster: failed to fetch dependency of "Cluster": failed to generate asset "Platform Quota Check": error(MissingQuota): ec2/L-0263D0A3 is not available in us-east-1 because the required number of resources (7) is more than the limit of 5, ec2/L-1216C47A is not available in us-east-1 because the required number of resources (18) is more than the limit of 5 
os:~ rappiti$ openshift-install destroy cluster Help me please

**Comment by vrutkovs** [comment, marked as answer]
>ec2/L-0263D0A3 is not available in us-east-1 because the required number of resources (7) is more than the limit of 5
>ec2/L-1216C47A is not available in us-east-1 because the required number of resources (18) is more than the limit of 5
